deploy/install.sh

```diff
@@ -117,9 +117,9 @@ applyDynaKubeCR() {
     apiUrl: ${API_URL}
     skipCertCheck: ${SKIP_CERT_CHECK}
 EOF
-  )
+  )"
 
-  classicFS=$(
+  classicFS="$(
     cat <<EOF
 
```

Comment:
Please remove all those unnecessary empty lines.

Reply:
They are necessary since otherwise when concatenating the strings the 2nd string would land on the same line as the first one

Reply:
Not if there's a proper linebreak at the end :-)

Reply:
Its due to how it is fumbled together. Which by the way is hard to understand.